README.md

```diff
@@ -108,53 +112,287 @@ ebcli rest-server --trust-node
 An api collection for [Postman](https://www.getpostman.com/) is provided [here](./docs/peggy.postman_collection.json) which documents some API endpoints and can be used to interact with it.
 Note: For checking account details/balance, you will need to change the cosmos addresses in the URLs, params and body to match the addresses you generated that you want to check.
 
-## Running the relayer service
+## Set up
+
+```bash
+cd testnet-contracts/
+
+# Create .env with sample environment variables
+cp .env.example .env
+```
+
+For running the bridge locally, you'll only need the `LOCAL_PROVIDER` environment variables. Environment variables `MNEMONIC` and `INFURA_PROJECT_ID` are required for using the Ropsten testnet.
+
+Further reading:
+
+- [MetaMask Mnemonic](https://metamask.zendesk.com/hc/en-us/articles/360015290032-How-to-Reveal-Your-Seed-Phrase)
+- [Infura Project ID](https://blog.infura.io/introducing-the-infura-dashboard-8969b7ab94e7)
+
+## Ethereum -> Cosmos asset transfers using the Bridge
+
+With the application set up, you can now use Peggy by sending a lock transaction to the smart contract.
+
+### Terminal 1: Start local blockchain
+
+```bash
+# Download dependencies
+yarn
+
+# Start local blockchain
+yarn develop
+```
+
+### Terminal 2: Compile and deploy Peggy contract
+
+```bash
+# Deploy contract to local blockchain
+yarn migrate
+
+# Copy contract ABI to go modules:
+yarn peggy:abi
+
+# Get contract's address
+yarn peggy:address
```

Comment:
Same here, all these additions were already in the contract tooling branch?

Reply:
`musnit/add-initial-burn-functionality` branch didn't have the contract tooling features yet, the new base branch does and this commit is no longer included in this PRs changes.